tgui/packages/tgui/interfaces/common/AtmosControls.tsx

```diff
@@ -73,6 +76,16 @@ export const Vent = (props: VentProps) => {
               })
             }
           />
+          <Button
+            icon={allow_automation ? 'check-square-o' : 'square-o'}
+            selected={allow_automation}
+            tooltip={`${allow_automation ? 'Disable' : 'Enable'} mode automation`}
+            onClick={() =>
+              act('automation', {
+                ref: refID,
+              })
+            }
+          />
```

Comment:
these should be more clear on what they do. Tooltips are insufficient for this, and the checkbox is not communicating "this will be locked" well at all. I don't see a reason you couldn't use text here? tho better icon choice might solve the problem just fine